lib/libutee/tee_api.c

```diff
@@ -202,12 +202,37 @@ TEE_Result TEE_Wait(uint32_t timeout)
 
 TEE_Result TEE_GetTAPersistentTime(TEE_Time *time)
 {
-	return utee_get_time(UTEE_TIME_CAT_TA_PERSISTENT, time);
+	TEE_Result res;
+
+	res = utee_get_time(UTEE_TIME_CAT_TA_PERSISTENT, time);
+
+	if (res != TEE_SUCCESS && res != TEE_ERROR_OVERFLOW) {
+		time->seconds = 0;
```

Comment:
@jenswi-linaro: "If an error other than TEE_ERROR_OVERFLOW is returned, this structure is filled with zeroes." What is your understanding about this ?